packages/teleport/src/components/Toggle/Toggle.tsx

```diff
@@ -17,10 +17,19 @@ limitations under the License.
 import React, { ReactNode } from 'react';
 import styled from 'styled-components';
 
-export default function Toggle({ isToggled, onToggle, children }: Props) {
+export default function Toggle({
+  isToggled,
+  onToggle,
+  children,
+  disabled,
+}: Props) {
   return (
-    <StyledWrapper>
-      <StyledInput checked={isToggled} onChange={() => onToggle()} />
+    <StyledWrapper disabled={disabled}>
```

Comment:
I don't think labels support the `disabled` attribute, but I'm okay with leaving this as it is for now.